micro_ros_diagnostic_bridge/src/diagnostic_bridge_node.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2018 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/microros/system_modes
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.hpp"
+
+#include <memory>
+
+int main(int argc, char ** argv)
```

Comment:
This file can be omitted completely by creating a component and node of MicroROSDiagnosticBridge using RCLCPP_COMPONENTS_REGISTER_NODE (in C++) with rclcpp_components_register_node (in CMake). See https://github.com/ros2/rclcpp/blob/master/rclcpp_components/cmake/rclcpp_components_register_node.cmake#L30 for details.